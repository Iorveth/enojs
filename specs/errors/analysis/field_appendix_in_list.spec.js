const eno = require('../../../eno.js');

const input = `
languages:
- eno
| json
`.trim();

describe('analysis.fieldAppendInList', () => {

  let error;

  beforeAll(() => {
    try {
      eno.parse(input);
    } catch(err) {
      error = err;
    }
  })

  it(`provides the correct message`, () => {
    expect(error.message).toMatchSnapshot();
  });

  it(`provides correct selection metadata`, () => {
    expect(error.selection).toEqual([[3, 0], [3, 6]]);
  });
});
