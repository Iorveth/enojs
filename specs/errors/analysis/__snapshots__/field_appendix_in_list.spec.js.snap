// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`analysis.fieldAppendInList provides the correct message 1`] = `
"Line 3 contains an appendix (to a field) inside a list.

   Line | Content
 *    1 | languages:
 *    2 | - eno
 >    3 | | json
"
`;
